euclidlike_imsim/noise.py

```diff
@@ -97,8 +97,15 @@ def get_noise(cfg_noise, cfg_image, base, logger):
     # Make integer ADU now.
     noise_img.quantize()
 
+    # Add quantization noise (this avoid issues related to rounding e.g.
+    # backgroud estimation)
```

Comment:
```suggestion
    # background estimation)
```

Reply:
Also, it could be good to give a link to the ERO paper and mention the section that discusses this, and maybe give a simple high-level summary, like "this makes it easier to process the images, without modifying the expectation value of the signal"